server/lib/pdf.js

```diff
@@ -36,9 +36,10 @@ export const getTransactionPdf = async (transaction, user) => {
 
 export const getConsolidatedInvoicesData = async fromCollective => {
   const transactions = await models.Transaction.findAll({
-    attributes: ['createdAt', 'HostCollectiveId', 'amountInHostCurrency', 'hostCurrency'],
+    attributes: ['createdAt', 'HostCollectiveId', 'amountInHostCurrency', 'hostCurrency', 'CollectiveId'],
     where: {
       type: 'CREDIT',
+      CollectiveId: { [Op.not]: fromCollective.id },
```

Comment:
Are we sure that this does not filter out refunds?

Reply:
> Are we sure that this does not filter out refunds?

@Betree I'm having trouble testing for individual refund locally, do you have any recommended way I can do that?

Reply:
Nvm, I found a way. 🙂

Reply:
I think this one is still an issue

Reply:
@Betree This does not filter out refund. I tested it.

Reply:
All good then, let's try it